workspace.go

```diff
@@ -829,6 +832,31 @@ func (s *workspaces) AddTagBindings(ctx context.Context, workspaceID string, opt
 	return response.Items, err
 }
 
+// DeleteTagBindings removes all tag bindings associated with a workspace. This
+// method will not remove any inherited tags.
+func (s *workspaces) DeleteTagBindings(ctx context.Context, workspaceID string) error {
+	if !validStringID(&workspaceID) {
+		return ErrInvalidWorkspaceID
+	}
+
+	type aliasOpts struct {
```

Comment:
No suggestions, just curious about the intent behind the name `aliasOpts`? What is "alias" referring to?

Reply:
I could have probably used a more descriptive name, but normally structs that "shadow" another -- for example to override struct tag definitions -- are referred to as aliases from what I've seen. 

A more descriptive name would be `aliasUpdateOptions`, but given this wasn't part of the public interface I decided for brevity. An alternative declaration could look like:
```go
opts :=  struct {
		Type        string        `jsonapi:"primary,workspaces"`
		TagBindings []*TagBinding `jsonapi:"relation,tag-bindings"`
	} {
		TagBindings: []*TagBinding{},
	} 
```

Let me know if you would prefer this change

Reply:
It's ok, no changes needed from my end.